magda-web-client/src/config.js

```diff
@@ -110,7 +110,8 @@ export const config = {
                             ? "mailto:data@digital.gov.au"
                             : "suggest"
                     ],
-                    ["Privacy Policy", "page/privacy-policy"]
+                    ["Privacy Policy", "page/privacy-policy"],
+                    ["Give feedback", "feedback"]
```

Comment:
This will mean it only shows on desktop, is that the plan?

Reply:
Mobile/small already had one of these links included in the config: https://github.com/TerriaJS/magda/blob/f5616ae1875cb27fc28adbf74955d0fd79a00c77/magda-web-client/src/config.js#L97